app/src/docs/v1.api-spec.yaml

```diff
@@ -211,6 +211,24 @@ components:
             field
           example:
             - baz@gov.bc.ca
+        cc:
+          type: array
+          items:
+            type: string
+          description: >-
+            An array of recipients email addresses that will appear on the cc:
+            field
+          example:
+            - baz@gov.bc.ca
+        bcc:
+          type: array
+          items:
+            type: string
+          description: >-
+            An array of recipients email addresses that will appear on the bcc:
+            field
+          example:
+            - baz@gov.bc.ca
```

Comment:
Will probably be good to have cc and bcc ordered alphabetically in the yaml spec - easier to find the properties/attributes down the road.
Also not a bad idea to use foo@gov.bc.ca and fizz@gov.bc.ca instead of baz@gov.bc.ca just to make sure there's some visual difference when ReDocs auto-generates the example.

Reply:
Done